### PR TITLE
Features/simplifywasm1

### PR DESCRIPTION
**Pull request by entlein**
Open for review and testing, will continue to make it better, but it *should* work even in this state. So if you do find any issues, do please report them

### CODE REVIEW

File: rke2/Makefile [hunk 1]
```diff
@@ -138,15 +138,10 @@ redpanda-baseline:
...
 .PHONY: redpanda-keys
 redpanda-keys:
-	#kubectl logs -n redpanda redpanda-src-0 -c redpanda |grep "transform - extractcsv" > redpanda/extractcsv/extract
-	#cut -f 11- -d ' ' redpanda/extractcsv/extract >> keys.csv
-	#cat -n keys.csv | sort -uk2 | sort -nk1 | cut -f2-  > uniquekeys.csv 
-	echo "do this next step manually while its a quick fix"
-	echo "gtimeout 20s kubectl --namespace redpanda exec -i -t redpanda-src-0 -c redpanda -- rpk topic consume extractcsv  | jq -r .key > uniquekeys.csv"
+	gtimeout 20s kubectl --namespace redpanda exec  redpanda-src-0 -c redpanda -- rpk topic consume extractcsv  | jq -r .key | tee uniquekeys.csv
 	awk '{printf "\"%s\",\n", $$0}' uniquekeys.csv > output.csv
 	paste -s -d ' ' output.csv > one_line_list.txt
```
Comment (pjoomen) [inline rke2/Makefile]: The parsing of the output from the `extractcsv` topic could be simplified by:
```
...|jq .key|sort -u|awk '{ print $$0","}'|paste -sd ' ' -|tee one_line_list.txt
```

File: rke2/Makefile [hunk 2]
```diff
@@ -138,15 +138,10 @@ redpanda-baseline:
 # for LINUX, use timeout instead of gtimeout
 .PHONY: redpanda-keys
 redpanda-keys:
-	#kubectl logs -n redpanda redpanda-src-0 -c redpanda |grep "transform - extractcsv" > redpanda/extractcsv/extract
-	#cut -f 11- -d ' ' redpanda/extractcsv/extract >> keys.csv
-	#cat -n keys.csv | sort -uk2 | sort -nk1 | cut -f2-  > uniquekeys.csv 
-	echo "do this next step manually while its a quick fix"
-	echo "gtimeout 20s kubectl --namespace redpanda exec -i -t redpanda-src-0 -c redpanda -- rpk topic consume extractcsv  | jq -r .key > uniquekeys.csv"
+	gtimeout 20s kubectl --namespace redpanda exec  redpanda-src-0 -c redpanda -- rpk topic consume extractcsv  | jq -r .key | tee uniquekeys.csv
```
[the remark this answers is not in the document]
Comment (pjoomen) [reply rke2/Makefile]: I now noticed the whole `redpanda-baseline` target was removed.

File: rke2/Makefile (hunk 1 above)
Comment (pjoomen) [reply rke2/Makefile]: Same as for previous comment.

**Comment by entlein** [comment]
As discussed in OfficeHours today, I ll merge this for now, and we explore our options on how to best deal with the dynamic baseline. We can expect a bit of breaking changes here, for now we focus on avoiding the GO-compile, until we have the next best solution

**Pull request merged**